go.mod

```diff
@@ -141,6 +141,3 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 	tags.cncf.io/container-device-interface/specs-go v0.7.0 // indirect
 )
-
-// Use the relative local source of the github.com/containerd/containerd/api to build
-replace github.com/containerd/containerd/api => ./api
```

Comment:
@dmcgowan Thinking of this in another way, we can always have a replace in the main branch/ development branch, but in release branches we maintain the correct tagged version for the API. WDYT?

Reply:
It should be fine to come and Go, but in practice the main branch will usually keep the replace, but every time we update it to a tag, it can go away. Normally that will be closer to release anyway. It will be good though to make sure we are testing and building our pre-releases from the tag though.

Reply:
Got it. In that case, once we have the `release-2.0`,  main will always have the `replace`